tools/distrib/python/grpcio_tools/setup.py

```diff
@@ -76,6 +77,10 @@
   if mac_target and (pkg_resources.parse_version(mac_target) <
 		     pkg_resources.parse_version('10.9.0')):
     os.environ['MACOSX_DEPLOYMENT_TARGET'] = '10.9'
+    os.environ['_PYTHON_HOST_PLATFORM'] = re.sub(
+        r'macosx-[0-9]+\.[0-9]+-(.+)',
+        r'macosx-10.9-\1',
+        util.get_platform())
```

Comment:
Missing `from distutils import util`? Should two files in this change be undergoing the exact same change?